cpp/examples/basic/CMakeLists.txt

```diff
@@ -1,12 +1,11 @@
-# Copyright (c) 2020-2024, NVIDIA CORPORATION.
+# Copyright (c) 2020-2025, NVIDIA CORPORATION.
 
 cmake_minimum_required(VERSION 3.26.4)
 
 include(../set_cuda_architecture.cmake)
 
 # initialize cuda architecture
 rapids_cuda_init_architectures(basic_example)
```

Comment:
Should look like the following so we still build for RAPIDS but still defers evaluation of `RAPIDS` to after project call
```
set(CMAKE_CUDA_ARCHITECTURES RAPIDS)
rapids_cuda_init_architectures(basic_example)
```

Reply:
This is already [the default in rapids-cmake](https://github.com/rapidsai/rapids-cmake/blob/branch-25.02/rapids-cmake/cuda/init_architectures.cmake#L83), but I'll go ahead and make it explicit here.

Reply:
Actually, going back on this since the proposed change would force building RAPIDS since users could not override. Making this overrideable while also setting the default is a bit more work, so let's skip this for now. We can always add more logic here if we need it.

Reply:
Implicitly using `RAPIDS` is fine with me.

Reply:
I assumed the intention of the original code was to hard force to RAPIDS no matter what the user requested. Since that looks to be wrong, happy to go with this :)